benchmarks/decoders/generate_readme_chart.py

```diff
@@ -56,7 +56,7 @@ def main() -> None:
         ffmpeg_path,
         videos_path,
     )
-    video_paths = glob.glob(f"{videos_path}/*.mp4")
+    video_files = glob.glob(f"{videos_path}/*.mp4")
```

Comment:
I was confused by how close the old variable names were to each other.

Reply:
See comment above about rename